src/controller/CHIPDeviceController.h

```diff
@@ -471,6 +490,14 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public SessionEst
 
 #endif
 
+    /**
+     * @brief
+     *   Register a pairing delegate to be notified of notable pairing events
+     *
+     * @param[in] pairingDelegate       Pointer to a DevicePairingDelegate.
```

Comment:
nit: this also seems redundant. Arguably the whole description comment may be extra: "Register Pairing Delegate" obviously "registers a pairing delegate". The only bit that maybe is not obvious is that this is for events only, however I am not sure if that information is that useful or will persist over time. If the delegate is for events only we could name it 'PairingEventsDelegate' and then comments would not be needed.